.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Not necessary.

Reply:
What exactly? If we keep only one, I would run on PR to make it easier to review PRs. As long as we don't allow direct pushes, we might then be able to get away without push.

Reply:
The line commented, `branches: [ master ]`.

Reply:
So you would prefer to run on all pull requests, not only those to master? Adding a bit more context instead of just a task would make it easier for me to follow along :)

Reply:
The pull requests will target only master.

Reply:
They're unlikely to happen, but sure.

Reply:
Ok, have changed it